deployment.go

```diff
@@ -99,6 +100,12 @@ func (d *Deployment) restartServices(services []string, failuresPermitted, timeo
 		return nil
 	}
 
+	// Don't restart services when a lock is present
+	if _, err := os.Stat("/etc/service/dont-restart"); err == nil {
```

Comment:
Sounds better, especially since its not like all `runit` things are not restarting or anything. Ill change